cloudinit/stages.py

```diff
@@ -252,12 +252,31 @@ def _get_data_source(self, existing):
             (cfg_list, pkg_list) = self._get_datasources()
```

Comment:
When I compare this commit to the original PR #229 ;  it removed the instance symlink;
```
 -           util.del_file(self.paths.instance_link)
```
Don't you need to also need to remove this bit?

Reply:
> When I compare this commit to the original PR #229 ; it removed the instance symlink;
> 
> ```
>  -           util.del_file(self.paths.instance_link)
> ```
> 
> Don't you need to also need to remove this bit?

I had removed this while testing but missed in my initial commit. I had removed this now